docs/source/running-autometa.rst

```diff
@@ -2,108 +2,271 @@
+
+*params.length_cutoff* : Smallest contiog you want binned (default is 3000bp)
+
+*params.kmer_size* : kmer size to use
+
+*params.kmer_norm_method* : Which normaization method to use. Choices are centered log-ratio transformation (clr), Autometa's clr implementation (am_clr) and isometric log-ratio transformation (default is am_clr)
+
+*params.kmer_pca_dimensions* : Number of dimensions to reduce the initial 512 dimension big k-mer matrix to (default is 50)
+
+*params.kmer_embed_method* :  kmer-embedding method. Choices are "sksne", "bhsne", "umap" (default is bhsne)
+
+*params.kmer_embed_dimensions* : Final dimensions you want to reduce the pca reduced kmer matrix to (default is 2).
+
+*params.kingdom* : Bins contigs belonging to which kingdom. Choices are "bacteria" and "archaea" (default is bacteria)
+
+*params.clustering_method* : Cluster contigs using which clustering method. Choices are "dbscan" and "hdbscan" (default is "dbscan")
+
+*params.binning_starting_rank* : Which taxonomic rank to start the binning from. Choices are "superkingdom", "phylum", "class", "order", "family", "genus", "species" (default is "superkingdom")
+
+*params.classification_method* : Which clustering method to use for unclustered recruitment step. Choices are "decision_tree" and "random_forest" (default is "decision_tree")
+
+*params.completeness* :  Minimum completeness needed to keep a cluster (default is atleast 20% complete)
+
+*params.purity* : Minimum purity needed to keep a cluster (default is atleast 95% pure)
+
+*params.cov_stddev_limit* : Which clusters to keep depending on the covergae std.dev (default is 25%)
+
+*params.gc_stddev_limit* : Which clusters to keep depending on the GC% std.dev (default is 5%)
+
+Running the pipeline
+--------------------
+
+You can run autometa using nextflow_ in multiple ways. You can install nexflow using conda by doing ``conda install -c conda-forge nextflow`` or you can also install from source using the instructions mentioned in their `documentation <https://www.nextflow.io/docs/latest/getstarted.html#installation>`_
+
+.. note::
+    1. Run the following commands directly in ``bash``. Nexflow will automatically submit jobs to SLURM or any other sheduling system.
+    2. You can use `tmux <https://github.com/tmux/tmux/wiki>`_ or `screen <https://www.gnu.org/software/screen/>`_ in case you want to exit the window or disconnect from the server.
+    3. The pipeline must be launched from a node where the ``condor_submit`` command is available, that is, in a common usage scenario, the cluster head node.
+    4. The HTCondor executor for Nextflow_ does not support at this time the HTCondor ability to transfer input/output data to the corresponding job computing node. Therefore the data needs to be made accessible to the computing nodes using a shared file system directory from where the Nextflow_ workflow has to be executed (or specified via the -w option).
```

Comment:
These are only relevant to users that want to run with an HTCondor executor. This should be moved to its own section. I think this would be intimidating to a new user trying out nextflow